src/components/headers/Default.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <header class="mdl-layout__header">
+    <div class="mdl-layout__header-row">
```

Comment:
No hamburger button for closing the sidebar for the desktop version yet; not a priority for the moment. Will do later.